lib/sda/overlay.go

```diff
@@ -234,12 +241,20 @@ func (o *Overlay) Close() {
 // CreateProtocol returns a fresh Protocol Instance with an attached
 // TreeNodeInstance
 func (o *Overlay) CreateProtocol(t *Tree, name string) (ProtocolInstance, error) {
```

Comment:
Same here regarding the method name, we should clearly distinguish between creating a protocol tied to a Service and creating a protocol which is independent. `CreateProtocol` is the default here but we don't know its behavior except if we look that there is also `CreateProtocolService`.